cms/djangoapps/contentstore/management/commands/tests/test_generate_test_course.py

```diff
@@ -0,0 +1,31 @@
+"""
+Unittest for generate a test course in an given modulestore
+"""
+import unittest
+import ddt
+from django.core.management import CommandError, call_command
+
+from contentstore.management.commands.generate_test_course import Command
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.django import modulestore
+
+
+@ddt.ddt
+class TestGenerateTestCourse(ModuleStoreTestCase):
+    """
+    Unit tests for creating a course in either old mongo or split mongo via command line
+    """
+
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_generate_course_in_stores(self, store):
+        call_command(
+            "generate_test_course", '{"store":"' + store + '","user":"' + str(self.user.email) + '","organization":"test-course-generator","number":"1","run":"1","fields":{"display_name":"test-course"}}'
```

Comment:
nit: This is hard to read. Maybe you can create a separate variable and then just add it into the call_command.

```
json_string = "Your really long nicely formatted json string"
call_command("generate_test_course", json_string)
```